jax_md/partition.py

```diff
@@ -544,6 +544,7 @@ def neighbor_list(displacement_or_metric: DisplacementOrMetricFn,
                   capacity_multiplier: float = 1.25,
                   disable_cell_list: bool = False,
                   mask_self: bool = True,
+                  custom_mask_function: Optional[Callable[[Array], Array]] = None,
```

Comment:
This line breaks the 80 char limit, is this okay?

Reply:
Hm, maybe we can define an alias `MaskFn = Callable[[Array], Array]`? (After this change I'm happy to merge, thanks for your patience!)

Reply:
Sounds good, I'll push that. 

It will then probably fail in linting, I need to make it conform before it'll be ready ready.